README.md

```diff
@@ -8,6 +7,12 @@ Creates the following resources:
 * Route53 record used to validate TLS certificate
 * Optional association with an ALB listener
 
+## Terraform Versions
+
+Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master branch.
+
+Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
```

Comment:
I think here we want to add the patch version too, as @brainsik pointed out in his review of the other module (so 2.0.0 and 1.0.0).